data/scrape/lib/generate_summary.py

```diff
@@ -38,12 +29,14 @@ def _handle_modifiers(
 def main() -> None:
     with open(LANGUAGES_PATH, "r", encoding="utf-8") as source:
         languages = json.load(source)
+
+    unique_iso_codes = set()  # To store unique ISO 639-2 codes
```

Comment:
Could you add type information here? I think you'll want:

```python
unique_iso_codes: Set[str] = set()  # Stores unique ISO 639-2 codes.
````

and so on for the following line.